src/components/pagination/pagination.stories.ts

```diff
@@ -27,15 +27,18 @@ export const simple = (): string => html`
   </calcite-pagination>
 `;
 
-export const darkModeFrenchLocale_TestOnly = (): string => html`<calcite-pagination
-  class="calcite-mode-dark"
-  start-item="1"
-  lang="fr"
-  group-separator
-  total-items="123456789"
-  page-size="10"
->
-</calcite-pagination>`;
+export const darkModeFrenchLocaleAndLargeScaleGetsMediumChevron_TestOnly = (): string => html`
+  <calcite-pagination
+    class="calcite-mode-dark"
+    start-item="1"
+    lang="fr"
+    group-separator
+    total-items="123456789"
+    page-size="10"
+    scale="l"
+  >
+  </calcite-pagination>
+`;
 
 darkModeFrenchLocale_TestOnly.parameters = { modes: modesDarkDefault };
```

Comment:
This needs to be changed to `darkModeFrenchLocaleAndLargeScaleGetsMediumChevron_TestOnly` as well, or else the story won't run in Chromatic